### DISCUSSION TITLE
Installation Issue: MOOSE Installation Fails on Linux Ubuntu 22.04.3 as libgcc-ng >=12 does not exist

### DISCUSSION THREAD

**Discussion by kurtenkera** [idaholab/moose#26822] [Q&A Getting Started] [answered]
I am a Linux user and I am trying to install MOOSE. I have closely followed the installation instructions on the MOOSE website (https://mooseframework.inl.gov/getting_started/installation/conda.html), but I get the following error message when I execute the command ` conda create -n moose moose-dev=2024.01.23` in my terminal:

``` (base) enk001@drift-ph:~$ conda create -n moose moose-dev=2024.01.23
Channels:
 - https://conda.software.inl.gov/public
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides libgcc-ng >=12 needed by moose-wasp-2023.11.29-build_0

Could not solve for environment specs
The following package could not be installed
└─ moose-dev 2024.01.23**  is not installable because it requires
   └─ moose-wasp 2023.11.29.* , which requires
      └─ libgcc-ng >=12 , which does not exist (perhaps a missing channel). 
```

It seems the issue is that `libgcc-ng >=12` does not seem to exist or is not detectable? Can anyone tell me how I can resolve this issue, as I desperately need to install MOOSE asap. Thanks in advance!

**Comment by milljm** [comment]
It might be that you are using an installation of Conda that does not include the `conda-forge` channel? Like Anaconda's main channel or something.

You might need to add conda-forge and then our channel:

```bash
conda config --add channels conda-forge
conda config --set channel_priority strict
conda config --add channels https://conda.software.inl.gov/public
```

- channel_priority strict will be necessary in your case I think
  - Ref: https://conda-forge.org/docs/user/tipsandtricks/#why-does-that-happen
- --add channel https://conda.software.inl.gov/public should be added again, to be at the top priority.

**Reply by kurtenkera** [reply, marked as answer]
Thanks very much. Adding conda-forge using `conda config --add channels conda-forge` worked. I didn't execute the other two commands you suggested as I tried this before your reply. But this still produced a successful installation. Thanks for your response!
